tsconfig.eslint.json

```diff
@@ -9,6 +9,7 @@
 		"./src/**/*.js",
 		"./src/**/*.ts",
 		"./src/**/*.svelte",
+		"./shared.utils.ts",
```

Comment:
`shared.utils` feels a bit like reading `utils.utils` 😄. What about `env.utils.ts` or `canister-ids.utils.ts`?

Reply:
`env.utils.ts` it is 👍